package.bat

```diff
@@ -27,22 +31,26 @@ cd ../rotkehlchen
 
 Rem Perform sanity checks that need pip install
 python -c "import sys;from rotkehlchen.db.dbhandler import detect_sqlcipher_version; version = detect_sqlcipher_version();sys.exit(0) if version == 4 else sys.exit(1)"
+@echo off
 if %errorlevel% neq 0 (
    echo "package.bat - ERROR: The packaging system's sqlcipher version is not >= v4"
    exit /b %errorlevel%
 )
 
 for /f %%i in ('python setup.py --version') do set ROTKEHLCHEN_VERSION=%%i
 
+@echo on
 Rem Use pyinstaller to package the python app
 IF EXIST build rmdir build /s /Q
 IF EXIST rotkehlchen_py_dist rmdir rotkehlchen_py_dist /s /Q
 pyinstaller --noconfirm --clean --distpath rotkehlchen_py_dist rotkehlchen.spec
+@echo off
```

Comment:
I am not sure what you are trying to achieve with removing echo at the checks? You find it confusing when Windows shows you the checks in the terminal? I am not opposed to this change, just want to understand it better. 

I personally find it easier to debug and understand what's happening when everything is printed.

Reply:
If you execute the .bat file while in an open terminal, it will actually print your if statements if echo is on, so you will get something like this:
![image](https://user-images.githubusercontent.com/27592/76201499-a4ea7f00-61f3-11ea-8a98-274360e2fb6c.png)

If you turn it off, you will get something like this:
![image](https://user-images.githubusercontent.com/27592/76201525-b0d64100-61f3-11ea-85c1-3d9d7b121de7.png)

When I see output like in the first screenshot where "error" is jumping out at me, I get confused. Ideally there should be echoes right before the if describing what the if does, and then a success/failure message. In this case we just have a comment and a failure message if it fails, which makes parsing the output of the package.bat for me a lot clearer (a success message could be added). Up to you!

Reply:
Sure okay that's fine for me. I see your point!